botocore/client.py

```diff
@@ -183,10 +183,13 @@ def _get_client_args(self, service_model, region_name, is_secure,
 
         endpoint_creator = EndpointCreator(self._endpoint_resolver,
                                            region_name, event_emitter)
+        timeout = (client_config.connect_timeout, client_config.read_timeout) \
```

Comment:
Bare bones, standard if statements are preferred.  They are the simplest and easiest to read, so that would be Option 1.